### ISSUE TITLE
Document new Dask behaviour and unpin its dependency from environment.yml

### ISSUE THREAD

**Issue by pgbarletta** [MDAnalysis/UserGuide#242]
Originally opened as a bug on the mdanalysis repo:  [#3931](https://github.com/MDAnalysis/mdanalysis/issues/3931).

I'll fix [this](section) after PR #241 is merged. Since this new behaviour forced me to pin Dask to an older version, I'll also unpit it once the notebook is fixed.

**Comment by orbeckst** [comment]
Thanks @pgbarletta .

Because you said that you'd address the issue I assigned it, but in principle you _should_ also be able to do so yourself. It's always good to see if someone has committed to taking charge of something.

**Comment by jandom** [comment]
@pgbarletta are you still interested in pushing forward this ticket? If it's not a priority any more, do you mind if we close out?

**Comment by pgbarletta** [comment]
> @pgbarletta are you still interested in pushing forward this ticket?

Yes. I have time to contribute again and I'm now going 1 issue at the time. This particular issue still needs to be addressed and I have to ask some questions to the coredevs before solving this.
